### PR TITLE
Colorize output for all terse formats

### LINKED ISSUE #1276: [FEATURE]: color output for all the terse formats
### Please describe your feature request

we should have coloring working with `buildtest rt --terse`, `buildtest buildspec find --terse` and all the options of **buildtest buildspec find** such as `buildtest --color=blue bc find -t --terse`.

Also have this working for `buildtest history list --terse`

### Post question in Slack

- [X] I agree that I posted my question in [slack](https://hpcbuildtest.slack.com/) before creating this issue

### Is there an existing issue

- [X] I confirm there is no existing [issue](https://github.com/buildtesters/buildtest/issues) for this issue

### PR DESCRIPTION
**Pull request by prathmesh4321**
Added color for terse option of  **buildtest buildspec maintainers --list** **buildtest buildspec maintainers --breakdown** along with **buildtest rt --terse** , **buildtest buildspec find --terse**, **buildtest history list --terse** 

image

image

image

image

image

image

### CODE REVIEW

File: buildtest/cli/history.py [hunk 1]
```diff
@@ -102,38 +102,20 @@ def list_build_history(no_header=None, terse=None, pager=None, color=None):
 
     if terse:
 
+        join_list = []
+
+        for key in table.keys():
+            join_list.append(table[key])
+
+        t = [list(i) for i in zip(*join_list)]
```
Comment (shahzebsiddiqui) [inline buildtest/cli/history.py]: perhaps use more meaningful variables for `t` i guess `transposed_list` i am not sure. For `join_list` perhaps use a different name perhaps `row_entry` or something i think that joined list is an entry or use a more meaningful name

File: buildtest/cli/history.py [hunk 2]
```diff
@@ -102,38 +102,20 @@ def list_build_history(no_header=None, terse=None, pager=None, color=None):
...
-            table["fail_tests"],
-            table["total_tests"],
-            table["pass_rate"],
-            table["fail_rate"],
-            table["command"],
-        ):
-            console.print(
-                f"{build_id}|{hostname}|{user}|{date}|{pass_test}|{fail_tests}|{total_tests}|{pass_rate}|{fail_rate}|{command}"
-            )
+        for i in t:
```
Comment (shahzebsiddiqui) [inline buildtest/cli/history.py]: perhaps change `i` to something like `row` or `line`

File: buildtest/cli/report.py [hunk 3]
```diff
@@ -590,10 +590,11 @@ def print_report(
                 t = t[:count]
 
             if not noheader:
-                print("|".join(self.display_table.keys()))
+                console.print("|".join(self.display_table.keys()), style=consoleColor)
 
             for i in t:
```
Comment (shahzebsiddiqui) [inline buildtest/cli/report.py]: same here perhaps use `row_entry` or `line` to indicate its a line

File: buildtest/cli/report.py [hunk 4]
```diff
@@ -590,10 +590,11 @@ def print_report(
...
-                print("|".join(i))
+                join_string = "|".join(i)
```
Comment (shahzebsiddiqui) [inline buildtest/cli/report.py]: rename this variable `join_string` perhaps `line` or `row_entry` or something else

**Review by shahzebsiddiqui — CHANGES_REQUESTED** [review]
a few minor changes to variable renaming.

**Comment by prathmesh4321** [comment]
> a few minor changes to variable renaming.

@shahzebsiddiqui Renamed the variables!

**Review by shahzebsiddiqui — APPROVED** [review]
thanks @prathmesh this looks better now

**Pull request merged**